apps/mull-ui/src/app/components/contact-row/contact-row.scss

```diff
@@ -5,49 +5,81 @@
   margin-top: 2rem;
   justify-content: space-between;
   align-items: center;
+
   .user-information {
     display: flex;
     flex-direction: row;
+    cursor: pointer;
   }
+
   .user-profile-picture {
     width: 3.5rem !important;
     height: 3.5rem !important;
     min-height: 3.5rem !important;
   }
+
   .user-details {
     display: flex;
     padding: 0.3rem;
     flex-direction: column;
     align-self: center;
+
     .username {
       margin-top: 0.5rem;
       font-weight: normal;
       font-size: 1.125rem;
       line-height: 1.125rem;
+      width: 100%;
+      max-width: 60vw;
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
     }
+
     .last-message {
       margin-top: 0.2rem;
       font-weight: 300;
       font-size: 0.75rem;
       line-height: 0.75rem;
+      width: 100%;
+      max-width: 60vw;
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
     }
+
     span {
       padding-left: 0.5rem;
     }
   }
   .contact-row-icon {
-    color: $primary-color;
-  }
-  a {
-    text-decoration: none;
-    color: black;
-    width: 100%;
+    .friend-settings-icon {
+      color: $primary-color;
+    }
+
+    a {
+      text-decoration: none;
+      color: black;
+      width: 100%;
+    }
+
+    a:hover {
+      cursor: pointer;
+    }
```

Comment:
The changes you did here broke the styling of the contact row in some places, Just move the `a` styling to be under .contact-container


![image](https://user-images.githubusercontent.com/23544999/114077319-a9112200-9875-11eb-8b07-2c0211cb80a0.png)